lib/common/UpdatingMercurial.md

```diff
@@ -0,0 +1,16 @@
+To update the mercurial zip files for the linux distributions you should perform the following steps or some reasonable facsimile:
```

Comment:
Is there anything similar to do for Windows? This file doesn't seem to be in a Linux-specific folder.